### ISSUE TITLE
frontend + backend

### ISSUE THREAD

**Issue by tony-lii** [dotref/paper-machine#9]
combine the frontend and backend stuff (raise new issue if app.py doesn't have enough functions)

First check with @huangrandy on the details of connection.
Make sure clicking button does not navigate the user to another page.

**Comment by SaltedFishLZ** [comment]
It is not "Done". It is closed because it is ILL-DEFINED.

**Comment by tony-lii** [comment]
![Image](https://github.com/user-attachments/assets/ab2f2703-de77-4d26-b163-10125dc42204)

@SaltedFishLZ can you clarify more on what would be expected for this story? I have attached a screenshot of what's currently done and connected. I think this story may be blocked until we have a more detailed design on ux.

**Comment by SaltedFishLZ** [comment]
By "ill-defined", I mean it should not be a stand-alone task.
It is definitely blocked by other progress since every time you update the frontend/backend, you need to do it again.
